apps/README.md

```diff
@@ -0,0 +1,14 @@
+[blockprotocol.org]: https://blockprotocol.org/?utm_medium=organic&utm_source=github_readme_blockprotocol-repo_apps
+[discord]: https://blockprotocol.org/discord?utm_medium=organic&utm_source=github_readme_blockprotocol-repo_apps
+[github_star]: https://github.com/blockprotocol/blockprotocol/tree/main/libs#
+[using blocks]: https://blockprotocol.org/docs?utm_medium=organic&utm_source=github_readme_blockprotocol-repo_apps
+[Þ docs]: https://blockprotocol.org/docs/using-blocks?utm_medium=organic&utm_source=github_readme_blockprotocol-repo_apps
+[`/libs/wordpress-plugin`]: /libs/wordpress-plugin
+
+[![discord](https://img.shields.io/discord/1050770647564943402)][discord] [![github_star](https://img.shields.io/github/stars/blockprotocol/blockprotocol?label=Star%20on%20GitHub&style=social)][github_star]
+
+# Apps
+
+This `apps` directory contains the source-code and content for the [blockprotocol.org] website. It will be expanded to include other "runnable" applications" as they become available.
```

Comment:
```suggestion
This `apps` directory contains the source-code and content for the [blockprotocol.org] website. It will be expanded to include other "runnable applications" as they become available.
```